nipype/interfaces/ants/legacy.py

```diff
@@ -94,16 +94,20 @@ class antsIntroduction(ANTSCommand):
 
     def _list_outputs(self):
         outputs = self._outputs().get()
+        transmodel = self.inputs.transformation_model
+
+        # When transform is set as 'RI'/'RA', no wrap fields will be outputed
+        if isdefined(transmodel) and transmodel not in ['RI', 'RA']:
```

Comment:
@lsqshr -  in that case the if statement should perhaps be:

`if not isdefined(transmodel) or (isdefined(transmodel) and transmodel not in ['RI', 'RA']):`